model/bindings.go

```diff
@@ -3,38 +3,35 @@ package model
 import (
 	"fmt"
 	log "github.com/Sirupsen/logrus"
-	yaml "github.com/cloudfoundry-incubator/candiedyaml"
 	"github.com/docker/libcompose/config"
-	"github.com/docker/libcompose/utils"
+	utils "github.com/docker/libcompose/utils"
+	libYaml "github.com/docker/libcompose/yaml"
 	"github.com/rancher/rancher-compose/preprocess"
-	"github.com/rancher/rancher-compose/rancher"
+	yaml "gopkg.in/yaml.v2"
 	"io/ioutil"
 	"os"
-	"reflect"
 )
 
-type mapLabel map[string]string
+type mapLabel map[string]interface{}
+type portArray []interface{}
 
 //BindingProperty holds bindings
 type BindingProperty map[string]interface{}
```

Comment:
Can we make this field a struct instead? It looks like the only field this should have is `Services`, which is a `map[string]ServiceBinding`.